core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt

```diff
@@ -20,4 +20,14 @@ class NavigationManager @Inject constructor(
             navigationCommandChannel.send(command)
         }
     }
+
+    fun back() {
+        externalMainImmediateScope.launch {
+            externalMainImmediateScope.launch {
```

Comment:
I think these scopes and coroutines are duplicated 🙂

Reply:
hey you're right, can't believe I miss this, thanks for spotting this 😃